src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -152,6 +147,16 @@ public string HubName
         // Used for mocking the lifecycle notification helper.
         internal HttpMessageHandler NotificationHandler { get; set; }
 
+        /// <summary>
+        /// Sets HubName to a value that is considered a default value.
+        /// </summary>
+        /// <param name="hubName">TaskHub name that is considered the default.</param>
+        public void SetDefaultHubName(string hubName)
```

Comment:
Does this need to be `public`, or can it be `internal`?

Reply:
I listed it as public so that any other DurabilityProviders can set the default as necessary, as we are currently doing in the AzureStorageDurabilityProviderFactory.